image-index.md

```diff
@@ -157,14 +156,9 @@ When the variant of the CPU is not listed in the table, values are implementatio
       }
     },
     {
-      "mediaType": "application/vnd.oci.artifact.manifest.v1+json",
+      "mediaType": "application/vnd.oci.image.index.v1+json",
       "size": 7682,
-      "digest": "sha256:601570aaff1b68a61eb9c85b8beca1644e698003e0cdb5bce960f193d265a8b7",
-      "artifactType": "application/example",
-      "annotations": {
-          "com.example.artifactKey1": "value1",
-          "com.example.artifactKey2": "value2"
-        }
```

Comment:
Will `artifactType` and `annotations` be dropped from projection into the descriptor if the items are artifacts? That's a regression vs OCI 1.1-rc.2 and Proposal E.

Implementations such as Homebrew currently rely on (unsafely!) using the `platform` field; annotations are more extensible.

Reply:
https://github.com/laboni999?tab=repositories

Reply:
The `artifactType` and `annotations` projections are more distribution-spec-isms than image-spec. I have not dropped the fields from `descriptor`.